### ISSUE TITLE
Consider providing the option to dynamically install the CMP

### ISSUE THREAD

**Issue by philpott** [legalwebio/gtm-template#1]
When reviewing the template we noticed that the template is only configuring consent settings. Have you considered also providing the option to install the CMP through the template using injectScript? This helps to simplify the installation of the CMP removing the necessity for adding any on-page code and simplifying consent mode setup.

**Comment by mredl** [comment]
Yes, because we don't have a "single install method". There are many ways how to integrate our CMP (native plugins, CDN, custom integration,...)
The injectScript approach would only help users which integrate via CDN, and to do this, it's only 2 lines which have to be copy pasted.
